base/applications/fontview/lang/pl-PL.rc

```diff
@@ -10,20 +11,20 @@ LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
 
 STRINGTABLE
 BEGIN
-    IDS_INSTALL "&Instaluj"
-    IDS_PRINT "&Drukuj"
+    IDS_INSTALL "Za&instaluj"
+    IDS_PRINT "Dr&ukuj"
     IDS_STRING "Zażółć gęślą Jaźń żółwiątkiem. 1234567890. !@#$%^&*()_+=-/?"
     IDS_OPEN "Otwórz czcionkę..."
     IDS_ERROR "Błąd"
-    IDS_ERROR_NOMEM "Brakuje pamięci do ukończenia tej operacji."
+    IDS_ERROR_NOMEM "Za mało pamięci do ukończenia tej operacji."
     IDS_ERROR_NOFONT "Plik %1 nie jest poprawnym plikiem czcionki."
     IDS_ERROR_NOCLASS "Nie udało się zainicjować klasy window."
-    IDS_ERROR_ISINSTALLED "This font is already installed!" 
-    IDS_ERROR_FONTCPY "Failed to copy the font file!"
-    IDS_ERROR_OPENKEY "Failed to open the fonts key!"
-    IDS_ERROR_REGISTER "Failed to register the new font!"
-    IDS_SUCCESS "Success"
-    IDS_COMPLETED "Font installation completed."
+    IDS_ERROR_ISINSTALLED "Ta czcionka jest już zainstalowana!" 
+    IDS_ERROR_FONTCPY "Nie można skopiować pliku czcionki!"
+    IDS_ERROR_OPENKEY "Nie można otworzyć pliku czcionki!"
```

Comment:
"fonts key" is not the same thing as "font file", and by grepping the code with `IDS_ERROR_OPENKEY` identifier, we can see that this error means that the system fonts registry key could not be opened (if it wasn't already clear from english string):
```
     /* Open the fonts key */
     res = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
                         L"SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts",
                        0,
                        KEY_ALL_ACCESS,
                        &hKey);
     if (res != ERROR_SUCCESS)
     {
         ErrorMsgBox(hwnd, IDS_ERROR_OPENKEY);
          return -1;
     }
```
```suggestion
    IDS_ERROR_OPENKEY "Nie można otworzyć klucza rejestru czcionek!"
```

Reply:
Indeed, the English wording of it is not very clear, I also made it explicit about the registry in ru-RU translation.